Backend/PersonDetection.Backend.Web/Hubs/VideoHub.cs

```diff
@@ -0,0 +1,38 @@
+using System.Diagnostics;
+using System.Threading.Channels;
+using Microsoft.AspNetCore.SignalR;
+using PersonDetection.Backend.Web.Services;
+
+namespace PersonDetection.Backend.Web.Hubs;
+
+public class VideoHub(IVideoPredictionsChannelService _predictionsChannelService) : Hub
+{
+    private static readonly string _sendModelPerformanceMethodName = "SendModelPerformance";
+    
+    private readonly Stopwatch _stopwatch = new Stopwatch();
+    
+    public async Task<ChannelReader<string>> ReceiveVideoData(string data)
+    {
+        _stopwatch.Start();
```

Comment:
This test code also gone in [Angular PR](https://github.com/jejikeh/maui-person-detection/pull/14)

I made a few small changes to the backend code while working on the frontend, but they were very minor. I didn't create a separate pull request for these changes, and the existing pull request is quite old, making it difficult to cherry-pick commits to this branch, I believe.